### PR TITLE
{math,vis}[GCCcore/11.3.0] OpenFOAM v2206, ParaView v5.10.1, CGAL v4.14.3, ... w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by kosl**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by kosl** [comment]
FFmpeg 5.0.1 is not compatible with ParaView 5.10.1 and that's why the latest v4 (4.4.2) is added too.

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.4.2-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://ffmpeg.org/releases/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['f98a482520c47507521a907914daa9efbc1384e0591b5afc3da18aa897de2948']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('pkg-config', '0.29.2')
```
Comment (maxim-masterov) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.4.2-GCCcore-11.3.0.eb:17]: Can you change pkg-config-0.29.2 to pkgconf-1.8.0?

Comment (kosl) [reply easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.4.2-GCCcore-11.3.0.eb]: Replaced with pkgconf

**Comment by kosl** [comment]
QuantumESPRESSO-7.1-foss-2022a.eb and some other packages is using older HDF5/1.12.2 while I have 1.13.1 and this is a problem?

**Comment by maxim-masterov** [comment]
@kosl yes, all HDF5 versions should be downgraded to 1.12.2, see PR #16153

File: easybuild/easyconfigs/p/ParaView/ParaView-5.10.1-foss-2022a-mpi.eb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    '59ca46a929a52d8abec107b72e19447cba3d8e64871b6fbc8e99b0f3b167db46',  # ParaView-v5.10.1.tar.gz
+]
+
+builddependencies = [('CMake', '3.23.1')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('XZ', '5.2.5'),
+    ('HDF5', '1.13.1'),
```
Comment (maxim-masterov) [inline easybuild/easyconfigs/p/ParaView/ParaView-5.10.1-foss-2022a-mpi.eb:29]: Can you also downgrade HDF5 here to `('HDF5', '1.12.2')`?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 13 out of 13 (11 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/0884beafacb8e8b52a713f518c185b96 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3151.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c53c4a0e0aa11195f76bfb576d118ec4 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @kosl!

**Comment by kosl** [comment]
Happy to get served.

**Pull request merged**